cli_test.go

```diff
@@ -571,3 +571,59 @@ func TestArgSetByUser(t *testing.T) {
 	}
 
 }
+
+func TestCommandAliases(t *testing.T) {
+	app := App("say", "")
+	app.ErrorHandling = flag.ContinueOnError
+
+	callCount := 0
+
+	app.Command("hello, hi", "", func(cmd *Cmd) {
+		cmd.Action = func() {
+			callCount++
+		}
+	})
+
+	err := app.Run([]string{"say", "hello"})
+	require.NoError(t, err, "should have returned without an error")
+	err = app.Run([]string{"say", "hi"})
+	require.NoError(t, err, "should have returned without an error")
+	err = app.Run([]string{"say", "hello, hi"})
+	require.Equal(t, callCount, 2, "command should have been invoked twice")
```

Comment:
this is not very readable IMHO: I had to re-read the code and debug the test because I thought this call executed the action twice.

Once easy fix is to do a table driven test with an expected err field set to false for the first 2  and to true for the last one, and have the app and command initialization done inside the for loop, this way they won't reuse the same callCount var (even better, a count is not needed anymore, a boolean is enough).